src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -38,6 +41,23 @@ public override JObject ToJson()
 
         public override bool Verify(DataCache snapshot, Transaction tx)
         {
+            // Check already stored signers
+
+            var conflictingSigners = tx.Signers.Select(s => s.Account);
+
+            foreach (var attr in tx.GetAttributes<Conflicts>())
+            {
+                var conflictRecord = snapshot.GetAndChange(new KeyBuilder(NativeContract.Ledger.Id, 11).Add(attr.Hash),
```

Comment:
if you can not use LedgerContract's field, please define 11 as a const value and use it here